cluster/rpc.go

```diff
@@ -113,9 +113,13 @@ type WritePointsRequest struct {
 
 // AddPoint adds a point to the WritePointRequest with field key 'value'
 func (w *WritePointsRequest) AddPoint(name string, value interface{}, timestamp time.Time, tags map[string]string) {
-	w.Points = append(w.Points, models.NewPoint(
+	pt, err := models.NewPoint(
 		name, tags, map[string]interface{}{"value": value}, timestamp,
-	))
+	)
+	if err != nil {
```

Comment:
Did you consider changing the return value of this function to support sending back an error?